addons/building_markers/functions/fnc_set.sqf

```diff
@@ -60,12 +60,16 @@ if (_set) then {
     _marker setMarkerSizeLocal _size;
     _marker setMarkerDir getDir _object;
     _object setVariable [QGVAR(marker), _marker, true];
+    missionNamespace setVariable [_marker, _object];
```

Comment:
I really would use the marker value in a "raw" format. I'm certain there are mods out there that already do that.

I would use something like this:
```suggestion
    missionNamespace setVariable [format [QGVAR(object_), _marker], _object];
```

I haven't checked if it works, but it would be safer to use imo.

Reply:
I should have just used netID.
_marker is already `format [QGVAR(%1), _object call BIS_fnc_netId]` `"zen_building_markers_0:1"`.
incidentally,
`allMapMarkers apply {isNil {missionnamespace getVariable _x}} // [true,true]`